godot/BasicDungeon/02.AndOrganic/AndOrganic.gd

```diff
@@ -78,15 +78,15 @@ func _add_room(rng: RandomNumberGenerator, data: Dictionary, rooms: Array, room:
 					rng.randf_range(rect.position.x, rect.end.x),
 					rng.randf_range(rect.position.y, rect.end.y)
 				))
-			poly_partial.sort_custom(self, "_lessv_x" if is_even else "_lessv_y")
+			poly_partial.sort_custom(Callable(self,"_lessv_x" if is_even else "_lessv_y"))
```

Comment:
Since in Godot v4 functions are callable by default, we don't need to create them with strings, unless we create the string dynamically based on some parameters.

This line should be: `poly_partial.sort_custom(_lessv_x if is_even else _lessv_y)`

Auto-completion also should help here instead of constructing with strings that are prone to typing-errors.